authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerHiveE2EIT.java

```diff
@@ -337,10 +350,80 @@ void testReadWriteTable() throws InterruptedException {
     // case 6: Fail to drop the table
     Assertions.assertThrows(AccessControlException.class, () -> sparkSession.sql(SQL_DROP_TABLE));
 
+    // case 7: If we don't have the role, we can't insert and select from data.
+    metalake.deleteRole(readWriteRole);
+    waitForUpdatingPolicies();
+    Assertions.assertThrows(AccessControlException.class, () -> sparkSession.sql(SQL_INSERT_TABLE));
+    Assertions.assertThrows(
+        AccessControlException.class, () -> sparkSession.sql(SQL_SELECT_TABLE).collectAsList());
```

Comment:
I think better to check all SQL operations.
1. SQL_CREATE_SCHEMA
2. SQL_USE_SCHEMA
3. SQL_CREATE_TABLE
4. SQL_INSERT_TABLE
5. SQL_SELECT_TABLE
6. SQL_UPDATE_TABLE
7. SQL_DELETE_TABLE
8. SQL_DROP_TABLE

Reply:
OK.